stacks/apps/dovetail-router.yml

```diff
@@ -674,8 +674,8 @@ Resources:
       Cluster: !Ref EcsClusterArn
       DeploymentConfiguration:
         MaximumPercent: 200
-        MinimumHealthyPercent: 50
-      DesiredCount: !If [IsProduction, 8, 2]
+        MinimumHealthyPercent: 100
+      DesiredCount: !If [IsProduction, 16, 2]
```

Comment:
I've noticed 2 things during deploys:

1. ECS tends to immediately kill half the DTR tasks, while launching new ones. That might be fine for other apps, but it can (right now) cause the remaining DTR tasks to be overwhelmed.
2. I've been manually changing the min-task count in autoscaling to 16.  But deploying keeps setting it back to 8, which is not ideal.